compiler/src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -109,6 +109,13 @@ object Formatting {
           case Atoms.Range(lo, hi) => CtxShow(s"Range(${toStr(lo.toList)}, ${toStr(hi.toList)})")
       end given
 
+      given Show[ast.untpd.Modifiers] with
+        def show(x: ast.untpd.Modifiers) =
+          CtxShow(s"Modifiers(${toStr(x.flags)}, ${toStr(x.privateWithin)}, ${toStr(x.annotations)}, ${toStr(x.mods)})")
```

Comment:
nit: Potentially this should be split off into a new PR?

Reply:
I've started to keep reasonable debugging changes in my PRs, rather than reapplying them locally every time or having to pull them into a dedicated PR. 😄